app/scripts/models/user.js

```diff
@@ -69,6 +69,14 @@ define(function (require, exports, module) {
       });
     },
 
+    /**
+     * Log the number of stored accounts
+     */
+    logNumStoredAccounts () {
+      const numAccounts = Object.keys(this._accounts()).length;
+      this._metrics.logEventOnce(`num.accounts.${numAccounts}`);
```

Comment:
Yeah agreed, this event should either use the .histogram feature of node-statsd or bucket the `numAccounts`